addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -339,18 +339,29 @@ var SnippetEditor = Widget.extend({
             return;
         }
 
+        const {transform} = targetEl.style;
+        if (transform) {
+            targetEl.style.transform = '';
+        }
```

Comment:
Doing it this way only works if the transform is inline (not coming from a CSS file) I guess?  Might want to improve that, that should not be too complicated using `getComputedStyle`.